js/script.js

```diff
@@ -1,205 +0,0 @@
-const startForm = {
-  // UI DECLARATION
-  ui: {
-    form: document.querySelector(".form"),
-    fields: document.querySelectorAll(".input-field"),
-    inputs: {
-      name: document.querySelector(".name"),
-      email: document.querySelector(".email"),
-      message: document.querySelector(".message")
-    },
-    button: document.querySelector(".button"),
-  },
-
-  // FUNCTIONS
-  functions: {
-    formValidation: () => {
-      const name = startForm.ui.inputs.name;
-      const email = startForm.ui.inputs.email;
-      const message = startForm.ui.inputs.message;
-      const input = startForm.ui.fields;
-      const button = startForm.ui.button;
-      let erros = 0;
-
-      const nameRegex = /[a-zA-Z\-'\s]+/;
-      const emailRegex = /^[A-z0-9\.\-]{1,}\@\w+\.[A-z]{2,3}(\.[a-z]{2})?$/;
-      const msgRegex = /.*\S.*/;
-
-      const regexValidation = (regexValue, input) => {
-        if (regexValue.test(input.value)) {
-          input.classList.remove("is-danger");
-          input.nextElementSibling.classList.add("is-hidden");
-        } else {
-          input.classList.add("is-danger");
-          input.nextElementSibling.classList.remove("is-hidden");
-          erros++;
-        }
-      };
-
-      regexValidation(nameRegex, name);
-      regexValidation(emailRegex, email);
-      regexValidation(msgRegex, message);
-
-      erros === 0 ? (button.disabled = false) : (button.disabled = true);
-    },
-
-    formMessage: status => {
```

Comment:
if and else is a better option in this block of code, since they belong to the same logic